spec/src/main/asciidoc/overview.asciidoc

```diff
@@ -170,6 +190,3 @@ org.eclipse.microprofile.example.ExampleBean/setCompletableFuture/1/ManagedExecu
 org.eclipse.microprofile.example.ExampleBean/appContext/ThreadContextConfig/cleared=Security,Transaction
 ----
 
-=== Thread Context Provider SPI
```

Comment:
Why did you drop this?

Reply:
Because we have an entire section dedicated to them later in the spec (in another file, but included nonetheless).